be/src/util/block_compression.cpp

```diff
@@ -816,38 +816,24 @@ class ZstdBlockCompression : public BlockCompressionCodec {
         return Status::OK();
     }
 
-    // follow ZSTD official example
-    //  https://github.com/facebook/zstd/blob/dev/examples/streaming_decompression.c
```

Comment:
add  some sql based regression test using zstd as compression method to ensure that the alg is right.

Reply:
ok, I will add a parquet zstd table regression test later.